manage.py

```diff
@@ -21,11 +22,13 @@ def test():
     # return the opposite boolean
     return not result.wasSuccessful()
 
+
 @app.route('/')
-def main():
+def main():  # pragma: no cover
```

Comment:
What's going on here? Is that commented out code or a comment?

Reply:
That tells coverage to skip the function